unit_tests/test_packages.py

```diff
@@ -214,9 +222,41 @@ def test_install_jenkins_bundle_download(self):
             hookenv.config()["bundle-site"] = "https://pkg.jenkins.io"
             bundle_path = os.path.join(hookenv.charm_dir(), "files")
             self.packages.install_jenkins()
-            self.assertTrue(glob.glob('%s/jenkins_*.deb' % bundle_path))
+            self.assertTrue(glob('%s/jenkins_*.deb' % bundle_path))
             self.assertEqual(
                 ["install"], self.fakes.processes.dpkg.actions["jenkins"])
         finally:
             hookenv.config()["release"] = orig_release
             hookenv.config()["bundle-site"] = orig_bundle_site
+    @mock.patch("charms.layer.jenkins.packages.os.system")
+    def test_clean_old_plugins(self, mock_os_system):
+        """
+        Make sure old plugin directories are excluded.
+        """
```

Comment:
No need for multiline here, can just do:

`"""Make sure old plugin directories are excluded."""`